### DISCUSSION TITLE
"Zero Horizon" landscape follows ecliptic instead of actual horizon now...

### DISCUSSION THREAD

**Discussion by ticowiko** [Stellarium/stellarium#3864] [General]
As of today, using "Zero Horizon" landscape mode doesn't actually cover up what's below the horizon. Instead the semitransparent circle follows the ecliptic exactly. Anyone else encountering this ?

**Comment by gzotti** [comment]
Is this suddenly in 24.2 (on which platform), or did you compile yourself? What GIT commit hash is that? I see no problem.

**Reply by ticowiko** [reply]
This is just the Android version, version 1.12.8, directly from the app store.

**Reply by gzotti** [reply]
OK, wrong place to report. This is for the desktop version only. Please see the contact listed in the app store.

**Reply by ticowiko** [reply]
Oh my bad ! Really sorry, didn't realize that. Will follow your advice then.
